pages/algorithms/equivalent-resource.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Equivalent resource 
+key: equivalent-resource 
+---
+
+Non-identical resources can still be equivalent by equally complying to the expectation formed by the user when navigating to them, thus serving an equivalent purpose. This would usually involve that the advertised key content is the same.  
+
+Web pages and documents (e.g. PDFs, office formats etc.) may fulfil be equivalent resources, even if the resources:
```

Comment:
There may be an extra word in this sentence, "may fulfil be equivalent resources".